conda_forge_tick/lazy_json_backends.py

```diff
@@ -634,14 +642,18 @@ def lazy_json_snapshot():
 
 
 @contextlib.contextmanager
-def lazy_json_override_backends(new_backends, hashmaps_to_sync=None):
+def lazy_json_override_backends(new_backends, hashmaps_to_sync=None, use_file_cache=None):
```

Comment:
Can we get a type annotation as well as a brief docstring about this parameter? To me, it is not immediately obvious.